app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,18 @@
+class CommentsController < ApplicationController
```

Comment:
# Optional: 
 - [ ] I will make this optional because your codes are working: 
 - Note: In order to create a functional form, you need two controller actions (and two routes as you did in the routes file). In Rails you will use: `new` and `create` actions for adding a new record to a database.
 - Therefore, I would advise you to add this code: 
 ```ruby
 def new
    @comment = Comment.new
  end
```